### ISSUE TITLE
Dynamics

### ISSUE THREAD

**Issue by grav** [SuneBear/Piano-Flow#5]
Would be nice to be able to control dynamics. Eg a scaling of the velocity by using the keys 1-9.

**Comment by jasononeil** [comment]
If using the mouse or touchpad the position of the vertical position of the click/touch appears to do what you want, I'm not sure what the most natural keyboard equivalent would be

**Comment by grav** [comment]
@jasononeil Yes, you're right - it does what I want. But it would be nice to be able to do this with the keyboard as well, as it makes for more rapid interaction (at least compared to a mouse).

Instead of pressing [any key], I'd press something between 1-9 (or maybe 1-0), and in addition to playing the current notes, it would also scale the velocity. Eg 5 being 50%.
